lib/Raven/Configuration.php

```diff
@@ -846,10 +813,20 @@ private function normalizeAbsolutePath($value)
             $path = $value;
         }
 
-        if (DIRECTORY_SEPARATOR === substr($path, 0, 1) && DIRECTORY_SEPARATOR !== substr($path, -1)) {
-            $path = $path . DIRECTORY_SEPARATOR;
+        if (DIRECTORY_SEPARATOR === $path[0] && DIRECTORY_SEPARATOR !== substr($path, -1)) {
+            $path .= DIRECTORY_SEPARATOR;
         }
 
         return $path;
     }
+
+    /**
+     * @param array $additionalOptions
+     */
+    private function mergeAndResolve(array $additionalOptions)
```

Comment:
I know I'm a fanatic of the documentation, but this is missing both the description of the method itself and of the argument